### DISCUSSION TITLE
Invoke-FalconRTR using incorrect path of pwsh.exe

### DISCUSSION THREAD

**Discussion by misteriks** [CrowdStrike/psfalcon#417] [General]
Invoke-FalconRTR -Command 'runscript' -Arguments '-CloudFile="some_powershell_script"' -HostIds $hostid -QueueOffline $true

If I execute this command locally it works and the Powershell script is executed successfully on the host

When I try to trigger the same command via Azure Functions the following exception is returned by Invoke-FalconRTR
The pwsh executable cannot be found at "C:\Program Files\Microsoft\Azure Functions Core tools\workers\powershell\7.2\runtimes\win\lib\net6.0\pwsh.exe"

Can I somehow instruct Invoke-FalconRTR to use the local host path for pwsh.exe ?

**Comment by bk-cs** [comment]
Is the target host producing that error, or Azure Functions? There's no code in PSFalcon that calls `pwsh.exe`. My guess is that this is the result of some environmental difference.
